src/cascadia/ut_app/TerminalApp.Unit.Tests.manifest

```diff
@@ -20,122 +20,4 @@
       <longPathAware xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">true</longPathAware>
     </windowsSettings>
   </application>
-  <!--

```

Comment:
I'm not sure why, but the `bcz` build itself seems to keep changing this file. Should I leave the changes? They seem similar to changes that were manually done in PR #2043 so I hope they're not bad.

Reply:
Yeah, I think it's fine. And that one is already merged. @DHowett-MSFT would be able to confirm.

Reply:
Oh shiz, that's actually a really good catch. I'll file a task to fix this the _right_ way. Basically #2043 needs to get applied to these tests too.

Reply:
(for know this is fine)